packages/fether-react/src/Feedback/Feedback.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import React from 'react';
+
+const feedbackUrl = 'https://github.com/paritytech/fether/issues/new';
+const openFeedbackLink = () => {
+  window.open(feedbackUrl, '_blank');
+};
+
+export const Feedback = () => (
+  <div className='feedback' onClick={() => openFeedbackLink()}>
```

Comment:
`onClick={openFeedbackLink)}` is slightly more performant

Reply:
I'm not sure what you mean? It's not defined in a class, so it doesn't work if I just do `onClick={openFeedbackLink()}` or `onClick={openFeedbackLink}